source/channels/join-leave-channels.rst

```diff
@@ -26,6 +26,10 @@ To join a public channel, select the **+** symbol at the top of the channel side
 
 To join a private channel, you need to be added to the channel by a member of that channel.
 
+.. tip::
+  
+  When you join new channels, depending on the channel actions configured, you may see a message welcoming you, and channels may be added to `a category in your channel sidebar <https://docs.mattermost.com/channels/customize-your-channel-sidebar.html>`__ automatically. Contact the Channel Admin for details.
```

Comment:
> When you join new channels, depending on the channel actions configured

Is it possible to link the sentence "the channel actions configured" to the paragraph that lists the automatic actions?

> Contact the Channel Admin for details

The user can see for themselves the actions configured for the channels they're a member of, even if only Channel Admins can modify them. They access in the same way as Channel Admins. Should we direct them to check for themselves instead of contacting the Channel Admin?

Reply:
These are great suggestions, @agarciamontoro! Thank you! I agree with and have implemented both.